src/test/java/org/trebol/operation/controllers/ControllersCorsTest.java

```diff
@@ -0,0 +1,235 @@
+						new ApiPath("/account/profile", "GET", "PUT"),
+						new ApiPath("/access", "GET"),
+						new ApiPath("/access/APIROUTE", "GET"),
+						new ApiPath("/data/people", "GET"),
+						new ApiPath("/data/billing_types", "GET"),
+						new ApiPath("/data/sell_statuses", "GET"),
+						new ApiPath("/data/customers", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/images", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/products", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/product_lists", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/product_list_contents", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/product_categories", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/sales", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/salespeople", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/shippers", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/users", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/user_roles", "GET", "POST", "PUT", "DELETE"));
+		
+		// Some ApiPaths with Invalid methods to test (if this fail, others should fail too)
+		apiPathsWithInvalidMethods = List.of(new ApiPath("/public/checkout", "GET"),
+											new ApiPath("/public/checkout/validate", "PUT"),
+											new ApiPath("/public/guest", "GET"),
+											new ApiPath("/public/login", "GET"),
+											new ApiPath("/public/register", "GET"),
+											new ApiPath("/data/sales/confirmation", "GET"),
+											new ApiPath("/data/sales/rejection", "GET"),
+											new ApiPath("/data/sales/completion", "GET"));
+	}
+	
+	// Arguments Providers
+	
+	// All paths with all methods and all allowed origins	
+	private Stream<Arguments> provideArgumentsWithAllowedOrigins() {
+		List<Arguments> args = new ArrayList<>();
+		
+		for (ApiPath apiPath : apiPaths) {
+			for (String method : apiPath.methods) {
+				for (String origin : allowedOrigins) {
+					args.add(arguments(apiPath.getPath(), method, origin));
+				}
+			}
+		}
+		
+		return args.stream();
+	}
+	
+	// All paths with all methods and all not allowed origins
+	private Stream<Arguments> provideArgumentsWithNotAllowedOrigins() {
+		List<Arguments> args = new ArrayList<>();
+		
+		for (ApiPath apiPath : apiPaths) {
+			for (String method : apiPath.methods) {
+				for (String origin : notAllowedOrigins) {
+					args.add(arguments(apiPath.getPath(), method, origin));
+				}
+			}
+		}
+		
+		return args.stream();
+	}
+	
+	// All paths with all methods and all allowed headers
+	private Stream<Arguments> provideArgumentsWithAllowedHeaders() {
+		List<Arguments> args = new ArrayList<>();
+		
+		for (ApiPath apiPath : apiPaths) {
+			for (String method : apiPath.methods) {
+				for (String header : allowedHeaders) {
+					args.add(arguments(apiPath.getPath(), method, header));
+				}
+			}
+		}
+		
+		return args.stream();
+	}
+	
+	// Some Paths with Invalid methods
```

Comment:
Please turn this into a method JavaDoc comment
```suggestion
	/** 
	 * Provides some Paths with invalid methods as test method parameters.
	 */
```